### ISSUE TITLE
Display warning when loading sinerider using non-standard URL (social first-play xp)

### ISSUE THREAD

**Issue by grymmy** [hackclub/sinerider#459]
### Description

We need to

1) Ensure that every time you are confirmed to have played the sinerider single-player game before, we save a browser cookie to persist that fact.
2) When sinerider's document::onload() is called, and IF the URL contains anything after the top-level domain (suggesting an entry path into sinerider that might be non-standard), check for the existence of the cookie in step #1, and if it does not exist:  a) set the cookie so the popup doesn't display again and b) display a modal popup with the following copy:

"New to Sinerider? You may want to start by playing the full game at sinerider.com!".

**Comment by grymmy** [comment]
@karamassie @polytroper Please approve the copy contained in this issue, I'd prefer having to do this once and only once.

**Comment by polytroper** [comment]
Yeah sounds great. Is it possible to have a direct link in the modal? Or perhaps we have one of the buttons on the modal open a new tab to sinerider.com?

**Comment by grymmy** [comment]
> Yeah sounds great. Is it possible to have a direct link in the modal? Or perhaps we have one of the buttons on the modal open a new tab to sinerider.com?

A direct link as in the URL contained within copy should be a hyperlink?  Yes that's fine.  Please elaborate on the button that you speak of, where should it display what copy on the button and what behavior should it have?
